lib/modules/subredditInfo.js

```diff
@@ -100,6 +104,8 @@ async function showSubredditInfo(card, subreddit) {
 			cacheFor: HOUR,
 		}): RedditSubreddit);
 	} catch (e) {
+		// Don't show pop up if no subreddit info
+		card.remove();
```

Comment:
Note that this makes `subredditInfoErrorLoadingSubredditInfo` (beneath) never appear.

I think this can also can be archived by simply removing `try...catch`, and Hover should automatically close the popup if an error is thrown.

Reply:
That is true. But I thought the purpose of the code would be clearer if I caught the AJAX exception.

Reply:
I think we do still want to display the error here if reddit is down / having api issues. Otherwise I think it'll be confusing if the popup just disappears.

But, hmm, do we need this line if we have the fake subreddit changes? Off the top of my head I can't think of any other error cases (sub names with invalid characters like `/r/res@tests` are handled by the regex, and valid names that don't exist return a search page, which is handled below). Maybe just remove this part for now?